### PR TITLE
 make html page

### PR DESCRIPTION
**Pull request by hibaakel**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Hiba's Angels Autocomplete</title>
+    <link rel="stylesheet" href="style.css">
```
Comment (mattlub) [inline public/index.html]: do you want to put href="./style.css" ? This is better in my opinion because it's more explicit

**Review by mattlub — CHANGES_REQUESTED** [review]

**Review by jwld — CHANGES_REQUESTED** [review]
Maybe we could add a placeholder for the input box. Do we need to have an empty value tag as well?

Might be good to add ids to both input and ul to make them easy to access from scripts later on.

**Review by MohamedOmarii — CHANGES_REQUESTED** [review]
I think the meta should be in the top under the meta charset and the title after the meta and the link .

**Review by MohamedOmarii — APPROVED** [review]

**Review by jwld — APPROVED** [review]

**Review by mattlub — APPROVED** [review]

**Comment by mattlub** [comment]
nice one

**Pull request merged**